### PR TITLE
added API 7 CropImageView compatibility

### LINKED ISSUE #2: update minsdkversion
The current minsdk is 14, but the project can compile against a minor api version.

### PR DESCRIPTION
**Pull request by cesards**
It closes #2 

Added API 7 support to CropImageView. It doesn't make sense to add API 1 version even if ImageView API hasn't changed at all.

Sorry about styles boilerplate. I was unconscious and young.

2  :+1:  needed

### CODE REVIEW

File: cropimageview/build.gradle [hunk 1]
```diff
@@ -1,14 +1,22 @@
 apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.jfrog.bintray'
+apply plugin: 'com.novoda.bintray-release'
```
Comment (malenalbc) [inline cropimageview/build.gradle]: What does this do exactly?

File: cropimageview/build.gradle [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 android {
-  compileSdkVersion 21
-  buildToolsVersion "21.1.2"
+  compileSdkVersion 23
+  buildToolsVersion "22.0.1"
 
   defaultConfig {
-    minSdkVersion 14
-    targetSdkVersion 21
+    minSdkVersion 7
```
Comment (malenalbc) [inline cropimageview/build.gradle]: I'm guessing the only real change is here right?

File: cropimageview/build.gradle (hunk 1 above)
Comment (cesards) [inline cropimageview/build.gradle]: It enables you uploading the lib to bintray + maven central (aar dependencies in our project)

File: cropimageview/build.gradle (hunk 2 above)
Comment (cesards) [inline cropimageview/build.gradle]: yep. But I had Dev broken... so I had to do some more changes...

Shame on me

**Comment by malenalbc** [comment]
:+1:

**Comment by juange87** [comment]
:+1:

**Pull request merged**